policy-controller/k8s/status/src/index.rs

```diff
@@ -74,14 +106,41 @@ impl Index {
         updates: UnboundedSender<Update>,
     ) -> SharedIndex {
         Arc::new(RwLock::new(Self {
-            _name: name.to_string(),
-            _claims: claims,
+            name: name.to_string(),
+            claims,
             updates,
             http_routes: HashMap::new(),
             servers: HashSet::new(),
         }))
     }
 
+    pub async fn run(index: Arc<RwLock<Self>>) {
+        // Clone the claims watch out of the index. This will immediately
+        // drop the read lock on the index so that it is not held for the
+        // lifetime of this function.
+        let mut claims = index.read().claims.clone();
+
+        loop {
+            tokio::select! {
+                _ = claims.changed() => {
+                    tracing::debug!("Lease holder has changed")
+                }
+                _ = time::sleep(Duration::from_secs(10)) => {}
+            }
+
+            // The claimant has changed or we should attempt to reconcile all
```

Comment:
very tiny grammar nit (sorry)
```suggestion
            // The claimant has changed, or we should attempt to reconcile all
```